### DISCUSSION TITLE
Feature Request - Update All

### DISCUSSION THREAD

**Discussion by bankenichi** [BlafKing/sd-civitai-browser-plus#109] [Ideas]
Rather than having to go through all models with available updates after scanning it would be great if there was an option to download all available updates with as few clicks as possible.

**Comment by BlafKing** [comment]
Hi there,

A similar request already exists, and unfortunately, it's tricky to implement.
I'm still looking into it though!

Please reference the previous feature request [here](https://github.com/BlafKing/sd-civitai-browser-plus/discussions/57) in the future for any updates regarding this feature :)
